_pages/en_US/installing-boot9strap-(frogtool).txt

```diff
@@ -0,0 +1,87 @@
+---
+title: "Installing boot9strap (Frogtool)"
+---
+
+{% include toc title="Table of Contents" %}
+
+{% capture technical_info %}
+<summary><em>Technical Details (optional)</em></summary>
+For technical details on the exploits that you will be using on this page, see "DSiWare_Exports CTCert verification" and "FIRM partitions known-plaintext" [here](https://www.3dbrew.org/wiki/3DS_System_Flaws).
+
+{% endcapture %}
+<details>{{ technical_info | markdownify }}</details>
+{: .notice--info}
+
+
+To use the [magnet](https://wikipedia.org/wiki/Magnet_URI_scheme) link for `frogcert.bin` on this page, you will need a torrent client like [qBittorrent](https://www.qbittorrent.org/download.php) or [Deluge](http://dev.deluge-torrent.org/wiki/Download).
+If you are unable to install one, follow [Bannerbomb3](bannerbomb3) instead.
```

Comment:
Is a new paragraph intended here? IIRC jekyll will parse this without a line break